cmd/access/node_builder/access_node_builder.go

```diff
@@ -572,29 +572,6 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 	})
 }
 
-// initMiddleware creates the network.Middleware implementation with the libp2p factory function, metrics, peer update
-// interval, and validators. The network.Middleware is then passed into the initNetwork function.
-func (builder *FlowAccessNodeBuilder) initMiddleware(nodeID flow.Identifier,
```

Comment:
The Stake AN and the UAN no longer share the common middleware that was being generated by this function. Instead they each have their own version of this method - one with peer manager and one without.